includes/functions/admin.php

```diff
@@ -375,3 +375,38 @@ class="button-primary button-large" target="_blank">
 		<?php
 	}
 }
+
+/**
+ * Notice to update the Pro addon if version is less then 1.1.2.
+ * This function can be remove June 1 2024.
+ * @since  3.1.43
+ *
+ * @return void
+ */
+function simcal_notice_to_update_pro_addon(){
+	$all_plugins = get_plugins();
+	$notices = get_option( 'simple-calendar_admin_notices', array() );
+
+	if ( array_key_exists( 'Simple-Calendar-Google-Calendar-Pro/simple-calendar-google-calendar-pro.php', $all_plugins ) &&  $all_plugins['Simple-Calendar-Google-Calendar-Pro/simple-calendar-google-calendar-pro.php']['Version'] > '1.1.2'  )  {
```

Comment:
Use https://www.php.net/manual/en/function.version-compare.php please

Reply:
The reason it is required is semver can not be compared like a number since it has more than one decimals

Reply:
And condition should be `<` , right?